scripts/install-helpers/baremetal-coco/intel-dcap/qgs.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: qgs-config
+  namespace: intel-dcap
+data:
+  qgs.conf: |
+    port = 4050
+    number_threads = 4
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sgx-default-qcnl-conf
+  namespace: intel-dcap
+data:
+  sgx_default_qcnl.conf: |
+    {
+      "pccs_url": "https://pccs-service:8042/sgx/certification/v4/",
+      "use_secure_cert": false,
+      "retry_times": 6,
+      "retry_delay": 10,
+      "pck_cache_expire_hours": 168,
+      "verify_collateral_cache_expire_hours": 168,
+      "local_cache_only": false
+    }
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: tdx-qgs
+  namespace: intel-dcap
+spec:
+  nodeSelector:
+    intel.feature.node.kubernetes.io/tdx: true
+  selector:
+    matchLabels:
+      app: tdx-qgs
+  template:
+    metadata:
+      labels:
+        app: tdx-qgs
+      annotations:
+        sgx.intel.com/quote-provider: tdx-qgs
+    spec:
+      hostNetwork: true
```

Comment:
is this a leftover from when we had PCCS serving localhost? I think this should be dropped.

Reply:
I fail to understand how this relates to SELinux. something to do with the `vsock` setup?

Reply:
I'm marking this one as resolved, as it doesn't make much sense spending time to improve this approach, as we'll be using the DCAP packages from Virt SIG soon enough.